### ISSUE TITLE
No DNS resolve request received with SLPS 2.0

### ISSUE THREAD

**Issue by afmklk** [Richard-Schaller/addon-syrlex2mqtt#3]
I followed the setup instructions but Dnsmasq logs do not show any domain resolve requests from my LEXplus 10S. It runs firmware SLPS 2.0, and is connected to local network via Wifi.

Is this the firmware issue mentioned in the documentation? Or do I first need to register the device with SYR Connect on the web or via their app to enable cloud services?

**Comment by afmklk** [comment]
After registering with SYR Connect, the device still connects to their servers (and is listed as „online“ in the SYR Connect App), even though I entered the local HA/Dnsmasq DNS server under network settings. Any idea how to fix this? Or is this because the device is running firmware 2.0?
![IMG_2951](https://github.com/user-attachments/assets/13dc1565-61db-4ce4-8258-4acd565a068b)

**Comment by Richard-Schaller** [comment]
I guess you already restarted the syr device after changing the settings? You should be able to see in the log of Dnsmasq what domains are queried (after a restart of the syr device). Maybe those are different compared to older firmware versions. There is likely more than one domain that is checked and it will get a connection to the syrconnect cloud if any of those works. If you do not see any request in the Dnsmasq log it looks like the network configuration of your LEXplus is wrong. You might check if your Dnsmasq is working at all by using the nslookup command line tool to request your Dnsmasq server to resolve the domain test.com (or any other domain):

`nslookup test.com 192.168.178.88`

Afterwards you should see an entry in the Dnsmasq log. If yes, you can look at the response of the redirected domains, e.g.

`nslookup syrconnect.de 192.168.178.88`

The problem described in the documentation is something different: In later firmware version you will not get a connection at all (neither to the official server not to syrlex2mqtt) as the https certificate must be signed (which is checked by the firmware).

**Comment by afmklk** [comment]
Thank you for your support! I did restart the Syr device (by unplugging power), but to no effect. 

Invoking the nslookup command for syrconnect.de from the Terminal on my Mac creates the following log entry in Dnsmaq:
```
dnsmasq[78]: query[A] syrconnect.de from 192.168.178.38
dnsmasq[78]: config syrconnect.de is 192.168.178.88
``` 

So Dnsmasq appears to be working, but the Syr device seems to ignore the network settings. I also noticed that they are greyed out and cannot be edited once I connect to the Wifi (see photo above).
